lib/iarc_v2/client.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf8 -*-
+import datetime
+import json
+import requests
+from urlparse import urljoin
+
+from django.conf import settings
+
+import commonware.log
+
+
+log = commonware.log.getLogger('z.iarc.v2')
+
+
+def get_rating_changes():
+    """
+    GetRatingChange for all changes within the last day.
+
+    TODO: Could add support for pagination, but very low priority since we
+          will never ever get anywhere close to 500 rating changes in a single
+          day.
+    """
+    if settings.IARC_V2_MOCK:
+        return json.loads(file('./mock/GetRatingChanges.json').read())
+    else:
+        start_date = datetime.datetime.utcnow()
+        end_date = start_date - datetime.timedelta(days=1)
+        url = urljoin(settings.IARC_V2_SERVICE_ENDPOINT, 'GetRatingChanges', {
+            'StartDate': start_date.strftime('%Y-%m-%d'),
+            'EndDate': end_date.strftime('%Y-%m-%d'),
+            'MaxRows': 500,  # Limit.
+            'StartRowIndex': 0  # Offset.
+        })
+        return requests.get(url).json()
+
+
+def search_certs(uuid):
+    """SearchCerts for complete data on a single certificate."""
+    if settings.IARC_V2_MOCK:
+        return json.loads(file('./mock/SearchCerts.json').read())
+    else:
+        url = urljoin(settings.IARC_V2_SERVICE_ENDPOINT, 'SearchCerts')
+        return requests.get(url, {'CertID': uuid}).json()
+
+
+def update_certs(cert_uuid, action):
```

Comment:
`cert_uuid` naming is not coherent with `search_certs()`.

Also, I don't really like the `action` parameter, because it exposes to the outside world the underlying implementation.